nixos/modules/appvm.nix

```diff
@@ -175,6 +175,21 @@ in
       }
     ];
 
+    security.polkit = {
+      enable = true;
+      extraConfig = ''
+        polkit.addRule(function(action, subject) {
+            if (action.id == "org.freedesktop.locale1.set-locale" && subject.user == "ghaf") {
```

Comment:
Should username be hardcoded? Or we should have `ghaf.username`  setting somewhere)

Reply:
Good question; I'll check.

Reply:
Better now?

Reply:
That broke standalone things, reverting for now.